src/github/client/github-anonymous-client.ts

```diff
@@ -61,4 +64,30 @@ export class GitHubAnonymousClient extends GitHubClient {
 		});
 	}
 
+	public async renewGitHubToken(opts: {
+		refreshToken: string,
+		clientId: string,
+		clientSecret: string
+	}) {
+		const { data: { access_token: accessToken, refresh_token: refreshToken } } = await this.axios.post(`/login/oauth/access_token`,
+			{
+				refresh_token: opts.refreshToken,
+				grant_type: "refresh_token",
+				client_id: opts.clientId,
+				client_secret: opts.clientSecret
+			},{
+				baseURL: this.baseUrl,
```

Comment:
I don't think you need to specify the baseUrl as it's already set in the parent `this.axios` instance

Reply:
`restApiURL` not `baseUrl` is set as axios base URL